Frontend/CO.CDP.OrganisationApp.Tests/Pages/Registration/OrganisationEntityFactory.cs

```diff
@@ -0,0 +1,50 @@
+using CO.CDP.Organisation.WebApiClient;
+
+namespace CO.CDP.OrganisationApp.Tests.Pages.Registration;
+public static class OrganisationEntityFactory
+{
+    public static Organisation.WebApiClient.Organisation GivenClientModel()
+    {
+        return new Organisation.WebApiClient.Organisation(null, null, null, Guid.NewGuid(), null, "Test Org", []);
+    }
+
+    public static ProblemDetails GivenProblemDetails(
```

Comment:
`GivenProblemDetails` and `GivenApiException` definitions seem to be duplicated in a couple of places. Any reason for this?

Reply:
I have prefixed with the called and the type of API has been linked to it.

Reply:
It was a confusion on my side. I didn't realise each web api client generates its own problem details type.